### DISCUSSION TITLE
Routes names for non English project

### DISCUSSION THREAD

**Discussion by arminhupka** [vercel/next.js#26398] [Help]
Hey, I’m coming here because I’m out of ideas how I should start.

I want create website which will be powered by Next. I don’t need any other languages than Polish so I don’t need some powerful plugins to handle localization. I’m not sure if I should create folders with polish names or make in other way?

**Comment by SalahAdDin** [comment]
Don't use polish names for localization, use english every time.

For localization you have a couple of packages like https://github.com/StarpTech/next-localization and https://github.com/isaachinman/next-i18next.

Look on internet, you will find a lot of good tutorials.
